app/controllers/reviews_controller.rb

```diff
@@ -161,6 +161,11 @@ def vote_down
       current_user.vote_against(@review)
     end
 
+    if ((@review.votes_against > -9) && @review.comment != "")
```

Comment:
they are displayed as negative numbers, but i'm not sure if `@review.votes_against` is a negative int or not, (which makes `>-9` suspicious)